### PR TITLE
adding storm_ref_filter.rstorm

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: docs/synapse/userguides/storm_ref_filter.rstorm [hunk 1]
```diff
@@ -0,0 +1,961 @@
...
+
+**Examples:**
+
+*Filter by Form (<form>):*
+
+- Downselect to include only domains:
+
+
+.. storm-pre:: [inet:fqdn=woot.com inet:fqdn=vertex.link  inet:fqdn=google.com inet:ipv4=127.0.0.1 ] +inet:fqdn
+.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_filter.rstorm:59]: We don't need to be sprinkling this directive around unless we're using it.

File: docs/synapse/userguides/storm_ref_filter.rstorm [hunk 2]
```diff
@@ -0,0 +1,961 @@
...
+    
+    <query> +inet:fqdn
+
+
+*Filter by Primary Property Value:*
+
+- Downselect to exclude the domain ``google.com``:
+
+
+.. storm-pre:: inet:fqdn=woot.com inet:fqdn=vertex.link inet:fqdn=google.com
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_filter.rstorm:70]: wouldn't we want this to be 
```
inet:fqdn=woot.com inet:fqdn=vertex.link inet:fqdn=google.com -inet:fqdn=google
```

File: docs/synapse/userguides/storm_ref_filter.rstorm [hunk 3]
```diff
@@ -0,0 +1,961 @@
...
+*<query* **+** | **-** *<form>* | *<prop>* ***range = (** *<range_min>* **,** *<range_max>* **)**
+
+**Examples:**
+
+*Filter by primary property in range:*
+
+- Downselect to include all IP addresses between 192.168.0.0 and 192.168.0.10:
+
+
+.. storm-pre:: [inet:fqdn=woot.com inet:fqdn=vertex.link inet:fqdn=google.com]
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_filter.rstorm:525]: how is this related to the filter?

File: docs/synapse/userguides/storm_ref_filter.rstorm [hunk 4]
```diff
@@ -0,0 +1,961 @@
...
+
+*<query>* **+** | **-** **#** *<tag>*
+
+*<query>* **+** | **-** **#** *<tag>* [ **:** *<tagprop>* [ *<operator>* *<pval>* ] ] 
+
+*<query>* **+** | **-** **#** *<tag>* **@=** *<time>* | **(** *<min>* **,** *<max>* **)**
+
+**Examples:**
+
+*Filter by tag:*
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_filter.rstorm:758]: Do we not have filters with tag globs documented?

File: docs/synapse/userguides/storm_ref_filter.rstorm [hunk 5]
```diff
@@ -0,0 +1,961 @@
...
+
+
+- From an initial set of ``file:bytes`` nodes, return only those that are detected as malicious by 10 or more AV vendors (i.e., that are associated with 10 or more ``it:av:filehit`` nodes):
+
+
+.. storm-pre:: file:bytes +{ -> it:av:filehit }>=10
+.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
+::
+
+    <file:bytes> +{ -> it:av:filehit }>=10
```
Comment (vEpiphyte) [inline docs/synapse/userguides/storm_ref_filter.rstorm:961]: please add a trailing newline :)

Comment (reign-git) [reply docs/synapse/userguides/storm_ref_filter.rstorm:961]: resolved

File: docs/synapse/userguides/storm_ref_filter.rstorm (hunk 2 above)
Comment (reign-git) [reply docs/synapse/userguides/storm_ref_filter.rstorm:70]: thanks good catch!

File: docs/synapse/userguides/storm_ref_filter.rstorm (hunk 3 above)
Comment (reign-git) [reply docs/synapse/userguides/storm_ref_filter.rstorm:525]: copy pasta oversight, fixed!

File: docs/synapse/userguides/storm_ref_filter.rstorm (hunk 1 above)
Comment (reign-git) [reply docs/synapse/userguides/storm_ref_filter.rstorm:59]: removed
